src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -203,7 +230,7 @@ IConfigurationSource configurationSource
     public class StateVectorToHtmlResultEncoder : IResultEncoder
     {
         private const double TWO_PI = 2.0 * System.Math.PI;
-
+        private double count = 0.0;
```

Comment:
A tip here: instead of adding a new `count` variable, you could edit this line below:
https://github.com/microsoft/iqsharp/blob/8589ea284a6ff48da5cf1f7438a8c424c85b18da/src/Jupyter/Visualization/StateDisplayEncoders.cs#L272
to become:
```
vector.SignificantAmplitudes(ConfigurationSource).Select((item, index) => 
```
and then `index` will be an integer that is automatically incremented for each iteration. No need to keep track of it yourself.

Reply:
I would be careful of duplicate divs with the same `id` if you go with this approach (i.e. multiple cells => `getElementById` might return the wrong one).

Reply:
I'm worried making this change will break how I've printed out all the percentages for numerical measurement probability display

Reply:
Makes sense! I didn't consider how this variable maintains its state across many calls to the encoder. I guess there may be other clever ways to do this, but this is fine as an initial implementation! (One more small suggestion: `count` could probably be an `int` rather than a `double`.)